service/src/main/java/com/zebrunner/reporting/service/util/TenancyInitializer.java

```diff
@@ -64,8 +64,9 @@ public void initTenancy(Message message) {
         try {
             EventMessage eventMessage = new Gson().fromJson(new String(message.getBody()), EventMessage.class);
             String tenancy = eventMessage.getTenantName();
-            LOGGER.info("Tenancy with name '" + tenancy + "' initialization is starting....");
```

Comment:
I know that we usually ignore this but lets try to use brackets placeholder `{}` for logging which is supported by slf4j